build-src.sh

```diff
@@ -3,23 +3,27 @@
 #
 # commit-db.rb list-valid nightly|GIT_DIR=/your/rust/dir/.git ./build-src.sh
 
+if [ $(uname) == 'Darwin' ]; then
+	alias tac='tail -f'
```

Comment:
Unless `tail -f` does something different on Mac than it does on GNU, I don't think this is correct. `tac` reverses all lines in the input.

Reply:
`man tail` says:

> The -r option causes the input to be displayed in reverse order, by line.  Additionally, this option changes the meaning of the -b, -c and -n options.  When the -r option is specified, these options specify the number of bytes, lines or 512-byte blocks to display, instead of the bytes, lines or blocks from the beginning or end of the input from which to begin the display.  The default for the -r option is to display all of the input.

Reply:
So I made a mistake, this should be `-r`, not `-f`.